k0764427/Assignment7.py

```diff
@@ -0,0 +1,18 @@
+from sys import argv
+
+script, text, surround = argv
```

Comment:
I would assume this is failing for you when run without arguments? Maybe only do this *if* you know there are arguments in `argv`